tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -0,0 +1,181 @@
+import os
+import rancher
+import time
+
+from .common import create_config_file
+from .common import create_user
+from .common import random_test_name
+from .common import readDataFile
+from .common import run_command_with_stderr
+from .common import set_url_password_token
+
+from lib.aws import AmazonWebServices
+
+DATA_SUBDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                           'resource')
+
+RANCHER_CHART_VERSION = os.environ.get("RANCHER_CHART_VERSION")
+test_suffix = random_test_name("auto-ha")
+temp_hostname = test_suffix + ".qa.rancher.space"
+# if hostname is not provided, generate one ( to support onTag )
+RANCHER_HA_HOSTNAME = os.environ.get("RANCHER_HA_HOSTNAME", temp_hostname)
```

Comment:
@bmdepesa Can the test_suffix be an environment variable that user can set and be used for both nlb and route53 entry that gets created ? ".qa.rancher.space" can be got as another environment variable .

Reply:
`qa.rancher.space` will never change as we use the hosted zone in Route53; I can parameterize the suffix and use that in place of the full host name (doing the concatenation in the test)

Reply:
Job now takes `RANCHER_HOSTNAME_PREFIX` and resources are now in format:
```
nlb-auto-21782-prefix
tg-80-auto-21782-prefix
tg-443-auto-21782-prefix
https://prefix.qa.rancher.space
```